PWGHF/TableProducer/candidateSelectorLc.cxx

```diff
@@ -235,6 +238,14 @@ struct HfCandidateSelectorLc {
       }
     }
 
+    /// cut on the Kpi pair invariant mass, to study Lc->pK*(->Kpi)
+    const double cutMassKPi = cuts->get(pTBin, "mass (Kpi)");
+    if (cutMassKPi > 0) {
+      if (std::abs(RecoDecay::m(std::array{trackKaon.pVector(), trackPion.pVector()}, std::array{o2::constants::physics::MassKaonCharged, o2::constants::physics::MassPiPlus}) - massK0Star892) > cutMassKPi) {
```

Comment:
I would add a function in the `hfHelper` for the massKPi calculation, as for the massPKPi, similar to those for the massKK of the Ds https://github.com/AliceO2Group/O2Physics/blob/d3b76b374b0aa86092f1bf70adf77eecffbc7b63/PWGHF/Core/HfHelper.h#L245-L255